dijet/tasks/plot_asymm.py

```diff
@@ -0,0 +1,149 @@
+# coding: utf-8
+
+import law
+
+# from dijet.tasks.base import HistogramsBaseTask
+from dijet.tasks.plot_base import PlottingBaseTask
+from columnflow.util import maybe_import, DotDict
+from columnflow.tasks.framework.base import Requirements
+
+from dijet.tasks.alpha import AlphaExtrapolation
+from dijet.constants import eta
+
+hist = maybe_import("hist")
+np = maybe_import("numpy")
+plt = maybe_import("matplotlib.pyplot")
+mplhep = maybe_import("mplhep")
+
+
+class PlotAsymmetries(PlottingBaseTask):
+    """
+    Task to plot all asymmetries.
+    One plot for each eta, pt and alpha bin for each method (fe,sm).
+    """
+
+    output_collection_cls = law.NestedSiblingFileCollection
+
+    # upstream requirements
+    reqs = Requirements(
+        AlphaExtrapolation=AlphaExtrapolation,
+    )
+
+    def create_branch_map(self):
+        """
+        Workflow has one branch for each eta bin (eta).
+        TODO: Hard coded for now
+              Into Base Task
+        """
+        return [
+            DotDict({"eta": (eta_lo, eta_hi)})
+            for eta_lo, eta_hi in zip(eta[:-1], eta[1:])
+        ]
+
+    def requires(self):
+        return self.reqs.AlphaExtrapolation.req(
+            self,
+            processes=("qcd", "data"),
+            branch=-1,
+        )
+
+    def load_asymmetry(self):
+        return (
+            self.input().collection[0]["asym"].load(formatter="pickle"),
+            self.input().collection[1]["asym"].load(formatter="pickle"),
+        )
+
+    def output(self) -> dict[law.FileSystemTarget]:
+        # TODO: Unstable for changes like data_jetmet_X
+        #       Make independent like in config datasetname groups
+        sample = self.extract_sample()
+        target = self.target(f"{sample}", dir=True)
+        # declare the main target
+        eta_lo, eta_hi = self.branch_data.eta
+        n_eta_lo = self.dot_to_p(eta_lo)
+        n_eta_hi = self.dot_to_p(eta_hi)
+        outp = {
+            "single": target.child(f"eta_{n_eta_lo}_{n_eta_hi}", type="d"),
+            "dummy": target.child(f"eta_{n_eta_lo}_{n_eta_hi}/dummy.txt", type="f"),
+        }
+        return outp
+
+    def plot_asymmetry(self, data, mc, asym):
+        fig, ax = plt.subplots()
+        plt.bar(
+            asym.flatten(),
+            mc.flatten(),
+            align="center",
+            width=np.diff(asym)[0],
+            alpha=0.6,
+            color="indianred",
+            edgecolor="none",
+            label="MC",
+        )
+        plt.scatter(asym.flatten(), data.flatten(), marker="o", color="black", label="Data")
+        ax.set_xlabel("A")
```

Comment:
Suggest to be more explicit here:
```suggestion
        ax.set_xlabel("Asymmetry")
```